pywt/tests/test_multilevel.py

```diff
@@ -447,5 +447,91 @@ def test_array_to_coeffs_invalid_inputs():
     assert_raises(ValueError, pywt.array_to_coeffs, arr, arr_slices, 'foo')
 
 
+def test_waverec_axes_subsets():
+    data = np.random.standard_normal((8, 8, 8))
```

Comment:
you still want to set a random seed here (and in tests below). the actual numbers don't matter, but a rare random failure is always possible with such tests.